static/app/views/settings/components/settingsNavigation.tsx

```diff
@@ -44,9 +42,7 @@ function SettingsSecondaryNavigation({
 
   return (
     <SecondaryNav>
-      <SecondaryNav.Header>
-        {PRIMARY_NAV_GROUP_CONFIG[PrimaryNavGroup.SETTINGS].label}
-      </SecondaryNav.Header>
+      <SecondaryNav.Header />
```

Comment:
Changed the behavior so that the header will by default use the default group label - otherwise the label would always have been settings